src/main/java/service/impl/FruitShopServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package service.impl;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+import java.util.List;
+import java.util.Map;
+import model.FruitOperation;
+import model.FruitType;
+import service.CalculateFruit;
+import service.FruitOperationParser;
+import service.FruitOperationTypeParser;
+import service.FruitShopService;
+import service.FruitTypeParser;
+import service.ReadDataFromFileSystem;
+import service.SaveToFileSystem;
+import service.SaveToStorage;
+
+public class FruitShopServiceImpl implements FruitShopService {
+    private final ReadDataFromFileSystem readDataFromFileSystem = new ReadDataFromFileSystemImpl();
+    private final CalculateFruit calculateFruitImpl = new CalculateFruitImpl();
+
+    private final FruitOperationTypeParser fruitOperationTypeParser
+            = new FruitOperationTypeParserImpl();
+    private final FruitTypeParser fruitTypeParser = new FruitTypeParserImpl();
+    private final FruitOperationParser fruitOperationParser
+            = new FruitOperationParserImpl(fruitOperationTypeParser, fruitTypeParser);
```

Comment:
too long and invlaid namings, classes and interfaces should not sound like an action
```suggestion
    private final Reader reader = new FileReader();
    private final FruitCalculator fruitCalculator = new FruitCalculatorImpl();

    private final FruitOperationTypeParser fruitOperationTypeParser
            = new FruitOperationTypeParserImpl();
    private final FruitTypeParser fruitTypeParser = new FruitTypeParserImpl();
    private final FruitOperationParser fruitOperationParser
            = new FruitOperationParserImpl(fruitOperationTypeParser, fruitTypeParser);
```